reana/reana_benchmark/analyze.py

```diff
@@ -321,7 +327,7 @@ def _save_plots(
 
 
 def analyze(
-    workflow_prefix: str, workflow_range: (int, int), plot_params: Dict
+    workflow_prefix: str, workflow_range: Tuple[int, int], plot_params: Dict
```

Comment:
Nicely solves the annotation error :+1: We should do the same to other annotations in this file, currently there are 7 errors. Could be done later..

Reply:
Initially, I wanted to amend everything but realized that I will touch code that doesn't belong to the PR/issue. If we are not strict about it, I can update the whole file. It will take a few minutes.

Reply:
I updated a couple of typings. In the next PRs, there are more fixes so will leave it as it is here.

Reply:
Done